avaframe/com1DFA/deriveParameterSet.py

```diff
@@ -41,14 +40,23 @@ def getVariationDict(avaDir, fullCfg, modDict):
         if key == 'resType':
             fullCfg = checkResType(fullCfg, section, key, value)
         # output saving options not relevant for parameter variation!
+        # percent variation info already used for updating thickness values
         if key not in ['resType', 'tSteps']:
             # if yes and if this value is different add this key to
             # the parameter variation dict
-            if ':' in value or '|' in value:
-                locValue = fU.splitIniValueToArraySteps(value)
+            if key in ['relThPercentVariation', 'entThPercentVariation', 'secondaryRelThPercentVariation'] and value != '':
+                # here the factor for changing thValues is added to the variationDict instead of the
+                # values directly
+                locValue = splitVariationToArraySteps(value, key, fullCfg)
                 variations[key] = locValue
                 defValue = modDict[section][key][1]
                 log.info('%s: %s (default value was: %s)' % (key, locValue, defValue))
+            else:
+                if ':' in value or '|' in value or '$' in value:
```

Comment:
if value in [':','|' ...
?

Reply:
I think this won't work as the it should check if in the value ':' is contained not if the value is ':'

Reply:
`if any(c in value for c in [':', '$', '|'])` then?

Reply:
changed